src/app/pages/fund/components/fiat-providers.utils.ts

```diff
@@ -68,7 +68,7 @@ function makeTransakUrl(address: string) {
 }
 
 function makeGenericExternalUrl(address: string, provider: string) {
```

Comment:
This isn't a great method name imo. This fn makes a url for the faq page with provider and address. 

`makeFundAccountProviderFaqUrl` ? 🤔

Reply:
Yeah, def better, I'll change.